components/teams-service/storage/postgres/postgres.go

```diff
@@ -179,29 +193,65 @@ func (p *postgres) EditTeam(ctx context.Context, team storage.Team) (storage.Tea
 
 // EditTeamByName edits the team's description and projects by name
 func (p *postgres) EditTeamByName(ctx context.Context,
-	teamName string, teamDescription string, teamProjects []string) (storage.Team, error) {
+	teamName string, teamDescription string, newProjects []string) (storage.Team, error) {
```

Comment:
this is not really `newProjects`, necessarily, since we don't have patchy updates. Maybe `updatedProjects`? or just `projects` (since we don't qualify the others such as `teamName`)